package/python-kiwi-spec-template

```diff
@@ -158,7 +156,6 @@ Requires:       zypper
 Provides:       kiwi-packagemanager:zypper
 %endif
 %if 0%{?debian} || 0%{?ubuntu}
-Requires:       debootstrap
```

Comment:
This needs to be changed to `Requires: apt`

Reply:
done